### PR TITLE
console app

### PR DESCRIPTION
**Pull request by LinaKrevetochka**
That's my version of the app. I just have to add Django. I wanted to add some screenshots, but I have a little problem when I predict mood of a new account. It also prints some warnings about GPU. I guess that's because I have trained neural network on Collab GPU.

### CODE REVIEW

File: Source/clean_data.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import re
+
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+
+
+def normalization(data):
+    data = data.str.lower()
+    data = data.apply(lambda tweet: re.sub(r"@[A-Za-z0-9]+", "", tweet))
```
Comment (Cjkjvfnby) [inline Source/clean_data.py]: Give names for these lambdas by extracting them to separate functions.

File: Source/clean_data.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+
+def lemmatizaton(data):
+    lemmatizer = WordNetLemmatizer()
+    return data.apply(
+        lambda tweet: [lemmatizer.lemmatize(word) for word in tweet]
+    )  # noqa
+
+
+def clean_data(x):
```
Comment (Cjkjvfnby) [inline Source/clean_data.py]: You could try something like that:

```python
processing_steps = [normalization, tokenization, delete_stop_words , ....]

return reduce(lambda acc, fun: fun(acc), processing_steps, text))

```

File: Source/get_prediction.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+def open_file(path):
+    with open(path, encoding="cp1252") as readfile:
+        df = pd.read_csv(readfile)
+    df = df[df["tweet"].notna()]
+    return df["tweet"], df["date"]
+
+
+def get_max_len(tweets):
```
Comment (Cjkjvfnby) [inline Source/get_prediction.py]: You can replace this function with https://docs.python.org/3/library/functions.html#max

File: Source/main.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+headline = """
+############################################################
+#                    Sentiment analysis                    #
+#                       with tweets                        #
+############################################################ """  # noqa
+
+
+def get_twits(name):
+    os.mkdir(f"../Users/{name}")
+    comand = f"twint -u {name} --since 2020-01-01 --until 2020-12-31 -o ../Users/{name}/{name}.csv --csv"
```
Comment (Cjkjvfnby) [inline Source/main.py]: Hardcoded year does not look good. Add it as a keyword argument with default value.

**Comment by Cjkjvfnby** [comment]
Please check if you have duplicated functions across files.

**Comment by LinaKrevetochka** [comment]
> Please check if you have duplicated functions across files.

I have some functions with equal names, that's because training and users tables are different. For example, training data has no column names and different coding. Should I rename them?

**Comment by Cjkjvfnby** [comment]
> I have some functions with equal names, that's because training and users tables are different. For example, training data has no column names and different coding. Should I rename them?

If they are different then ok.  Renaming is optional because we will never use that in that same context.

**Pull request merged**